templates/Dockerfile_ui2.template

```diff
@@ -2,33 +2,16 @@ FROM ubuntu:18.04 as builder
 
 RUN apt update \
     && apt install -yq \
-    openjdk-11-jre \
-    git \
     wget \
     && apt clean
 
-# install nvm
-RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
-
-# cache busting in order to get the up-to-date Git repo
-RUN touch {{ DOCKSTORE_UI2_VERSION }}.out
-
-RUN git clone https://github.com/dockstore/dockstore-ui2.git
-WORKDIR /dockstore-ui2
-RUN git checkout {{ DOCKSTORE_UI2_VERSION }}
-
-# install node version from .nvmrc file
-run bash -i -c 'nvm install'
-
-# inherit $CI in order to suppress Cypress TTY output when running on CI server
-ARG CI
-ENV CI $CI
-RUN bash -i -c 'npm ci'
-
-# build
-RUN bash -i -c 'NODE_OPTIONS="--max-old-space-size=4096" npm run build.prod'
+# Download index.html for UI we will be serving
+RUN wget -qO- https://gui.dockstore.org/{{ DOCKSTORE_UI2_VERSION }}/index.html
 
+# Insert full qualified paths.
+RUN sed -i 's/manifest.json/https:\/\/gui.dockstore.org\/{{ DOCKSTORE_UI2_VERSION }}/manifest.json/' index.html
+RUN sed -i 's/styles\./https:\/\/gui.dockstore.org\/{{ DOCKSTORE_UI2_VERSION }}/styles\./' index.html
+RUN sed -i 's/src="/src="https:\/\/gui.dockstore.org\/{{ DOCKSTORE_UI2_VERSION }}\//' index.html
 FROM nginx:1.13.1
 
-COPY --from=builder /dockstore-ui2/dist /usr/share/nginx/html2
-COPY config/{{ GOOGLE_VERIFICATION_NAME }}.html /usr/share/nginx/html2/{{ GOOGLE_VERIFICATION_NAME }}.html
+COPY --from=builder index.html /usr/share/nginx/html2
```

Comment:
This creates a folder called index.html

Reply:
>     1. ~Needed to set CORs on the S3 bucket~ (CORS set on the S3 bucket don't seem to fix manifest.json issue)

Maybe we should just copy manifest.json as well into the nginx container and not change the path.

>        a) resolution: set cors on bucket, also some cloudfront config https://aws.amazon.com/premiumsupport/knowledge-center/no-access-control-allow-origin-error/
> 
>     2. ~MIME types not set by CircleCI upload~
>        a) Crummy [cargomedia/cm#1712 (comment)](https://github.com/cargomedia/cm/issues/1712#issuecomment-88882940)
>        b) resolution: moved along with index.html

I think/thought CF should handle this. I wonder if it's because of the whitelisting of headers that you added. Could we roll that back and if we serve up the manifest.json from the Dockstore VM, or were there other files that had CORS issues?
> 
> Future work:
> 
>     1. image locations all messed up
> 
>     2. Some fonts seem to be loading as the index.html (more paths)?
> 
>     3. Scripts too, there are more internal references between scripts presumably (ex: common.feac484f496a5613204d.js loaded as index.html)

Reply:
> Maybe we should just copy manifest.json as well into the nginx container and not change the path.

Yup, that's the approach I took in the hack below which needs to be merged

> I think/thought CF should handle this. I wonder if it's because of the whitelisting of headers that you added. Could we roll that back and if we serve up the manifest.json from the Dockstore VM, or were there other files that had CORS issues? 

Pretty much all script files had issues at least